### PR TITLE
dart: add page

### LINKED ISSUE #6349: page request: dart
https://dart.dev/tools/dart-tool

### PR DESCRIPTION
**Pull request by pixelcmtd**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

closes #6349

### CODE REVIEW

File: pages/common/dart.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# dart
+
+> The tool for managing Dart projects.
+> More information: <https://dart.dev/tools/dart-tool>.
+
+- Create a new project:
```
Comment (marchersimon) [inline pages/common/dart.md:6]:
```suggestion
- Create a new project in the current directory:
```

Comment (chrissxYT) [reply pages/common/dart.md:6]: It doesn't. It creates a new directory containing the new project.

Comment (CleanMachine1) [reply pages/common/dart.md:6]: where?

Comment (chrissxYT) [reply pages/common/dart.md:6]: `./{{project_name}}`

Comment (marchersimon) [reply pages/common/dart.md:6]: Hmm, that's not so easy to say in one sentence. The only thing I can come up with:
> Create a new project direcory in the current directory:

Comment (blueskyson) [reply pages/common/dart.md:6]: How about:
> Create a new project directory in current path

Comment (chrissxYT) [reply pages/common/dart.md:6]: Well, that seems kinda confusing to me, because it makes it sound like it *just* creates a directory.

Comment (navarroaxel) [reply pages/common/dart.md:6]: I vote to leave the description as-is.

Comment (marchersimon) [reply pages/common/dart.md:6]: Sounds like the best option.

**Review by marchersimon — APPROVED** [review]
Looks good, thanks

File: pages/common/dart.md (hunk 1 above)
Comment (sbrl) [reply pages/common/dart.md:6]: I like @marchersimon's suggestion here of `Create a new project directory in the current directory:`.

Comment (glapa-grossklag) [reply pages/common/dart.md:6]: "Create a directory in the current directory for a new project"?

Just another idea.

**Review by navarroaxel — APPROVED** [review]

File: pages/common/dart.md (hunk 1 above)
Comment (navarroaxel) [reply pages/common/dart.md:6]: @pixelcmtd what do you think?
```suggestion
- Initialize a new Dart project in a directory of the same name:
```

Comment (marchersimon) [reply pages/common/dart.md:6]: I think we could leave it as is, but I'm fine with this one also.

Comment (pixelcmtd) [reply pages/common/dart.md:6]: > @pixelcmtd what do you think?

I personally really don't like unnecessarily long descriptions and this whole review seems to me like a competition on who can come up with the longest, worst-sounding one.

**Review by einverne — APPROVED** [review]
Looks good.

**Pull request merged**